packages/spacecat-shared-rum-api-client/src/functions/form-vitals.js

```diff
@@ -143,17 +167,30 @@ function handler(bundles) {
   const formViewdataChunks = new DataChunks();
   loadBundles(bundlesWithFilteredEvents, formViewdataChunks);
   const formSourceMap = {};
+  const iframeParentMap = {};
   const globalFormSourceSet = new Set();
-  formViewdataChunks.filter = { checkpoint: ['viewblock'] };
+  formViewdataChunks.filter = { checkpoint: ['viewblock', 'viewmedia'] };
   formViewdataChunks.filtered.forEach(({ url, events }) => {
     formSourceMap[url] = formSourceMap[url] || new Set();
-    events.forEach(({ checkpoint, source }) => {
+    events.forEach(({ checkpoint, source, target }) => {
       if (checkpoint === 'viewblock' && source) {
         formSourceMap[url].add(source);
         globalFormSourceSet.add(source);
       }
+      if (checkpoint === 'viewmedia' && target) {
+        if (target.includes('aemform.iframe')) {
```

Comment:
can you pls update this to include cases like: `https://www.micron.com/content/micron/us/en/forms/contact-us/jcr:content/root/responsivegrid/aemform_867354491.iframe.en.html`